docs/application/dotnet/guides/overview.md

```diff
@@ -32,9 +37,9 @@ You can use the following TizenFX features in your .NET applications:
   The Wearable Circular UI feature introduces a set of helpful extensions of the <a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/" target="_blank">Xamarin.Forms</a> framework.
   Its aim is to develop an open source software and to motivate software developers to create Tizen Wearable Xamarin.Forms application more easily and efficiently.
 
-- [Localization](internationalization/localization.md)
+- [Internationalization](internationalization/phonenumber.md)
```

Comment:
Please don't change the previous updates and just update OpenTK contents in this file.